src/audio/copier/copier_host.c

```diff
@@ -4,6 +4,87 @@
 //
 // Author: Baofeng Tian <baofeng.tian@intel.com>
 
+#include <sof/audio/component_ext.h>
 #include <sof/trace/trace.h>
+#include <ipc4/copier.h>
+#include <sof/audio/host_copier.h>
```

Comment:
move "include <sof/audio/host_copier.h>" to above and follow "#include <sof/audio/component_ext.h>"